content/messaging/agent-chat-api/v3.3/index.mdx

```diff
@@ -511,7 +511,7 @@ It returns [summaries](#chat-summaries) of the chats an Agent has access to.
 
 | Parameter                                             | Required | Type     | Notes                                                                                       |
 | ----------------------------------------------------- | -------- | -------- | ------------------------------------------------------------------------------------------- |
-| `filters`                                             | No       | `object` | Mustn't change between requests for subsequent pages. Otherwise, the behavior is undefined. |
+| `filters`                                             | No       | `object` |                                                                                             |
```

Comment:
```suggestion
| `filters`                                             | No       | `object` | When paginating, `filters` provided in the first request are remembered and automatically used for the subsequent requests. Providing a new `filters` object will result in a `validation` error.  To reset the filters, start paginating with a new set of filters.|
```

Reply:
WDYT, @kacperf531?

Reply:
sure, commited your suggestion, sorry for the delay :(